web/modules/custom/paragraph_behavior/src/Plugin/paragraphs/Behavior/ImageBehavior.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\paragraph_behavior\Plugin\paragraphs\Behavior;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\paragraphs\Entity\ParagraphsType;
+use Drupal\paragraphs\ParagraphInterface;
+use Drupal\paragraphs\ParagraphsBehaviorBase;
+
+/**
+ * @file
+ * Additional behavior for paragraph "Photo with text" type.
+ */
+
+/**
+ * Class provides image behavior for "Photo with text" paragraph type.
+ *
+ * Setting can be enabled on paragraph type configuration page.
+ *
+ * @ParagraphsBehavior(
+ *  id = "paragraphs_behavior",
+ *  label = @Translation("Image settings"),
+ *  description = @Translation("Settings for photo with text paragraph type"),
+ *  weight = 0,
+ * )
+ */
+class ImageBehavior extends ParagraphsBehaviorBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function isApplicable(ParagraphsType $paragraphs_type) {
+    return $paragraphs_type->id() == "photo_with_text";
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function view(array &$build, Paragraph $paragraphs_entity, EntityViewDisplayInterface $display, $view_mode): void {
+    $image_position = $paragraphs_entity->getBehaviorSetting($this->getPluginId(), 'image_position');
```

Comment:
missing 3 parameter for default value

Reply:
Do i need change it like this ?

`... getBehaviorSetting($this->getPluginId(), 'image_position'**, 'left'**);`

Reply:
I am not sure, dose 3 means three or third